algorithm/marine/soca_vtscales.yaml.j2

```diff
@@ -1,6 +1,6 @@
-gridspec_filename: soca_gridspec.nc
-restart_filename: ./INPUT/MOM.res.nc
-mld_filename: './staticb/ocn.bkgerr_stddev.incr.{{ marine_window_end_iso }}.nc'
+gridspec_filename: ./anl_geom/soca_gridspec.nc
+restart_filename: './anl_geom/ocn.f009_anl_geom.fc.{{ marine_window_end_iso }}.PT0S.nc'
+mld_filename: './anl_geom/ocn.f009_anl_geom.fc.{{ marine_window_end_iso }}.PT0S.nc'
```

Comment:
for my education: what does this change mean?

Reply:
To prepare the vertical localization scales at low res. I'm also not sure why the mld was stored in the bkg err file, this was a mistake. Does that answer your question?